framework/src/Platform.ts

```diff
@@ -107,7 +107,7 @@ export abstract class Platform<
         dbPlugin.config.storedElements = dbPlugin.getDefaultConfig().storedElements || {};
       }
       // eslint-disable-next-line no-console
-      console.warn(`Session storage was enabled for database plugin ${dbPlugin.constructor.name}`);
+      console.warn(`Session storage was enabled for database plugin ${dbPlugin.name}`);
```

Comment:
Needs to be updated to `Logger.warn` after JovoLogger is merged.

Reply:
I will do that in a separate PR but thanks for the reminder!